lstchain/data/lstchain_standard_config.json

```diff
@@ -133,74 +133,54 @@
     "class_weight": null
   },
 
+
   "energy_regression_features": [
     "log_intensity",
     "width",
     "length",
-    "x",
-    "y",
-    "psi",
-    "phi",
     "wl",
     "skewness",
     "kurtosis",
-    "r",
     "time_gradient",
     "leakage_intensity_width_2"
   ],
 
+  "disp_method": "disp_norm_sign",
+
   "disp_regression_features": [
     "log_intensity",
     "width",
     "length",
-    "x",
-    "y",
-    "psi",
-    "phi",
     "wl",
     "skewness",
     "kurtosis",
-    "r",
     "time_gradient",
     "leakage_intensity_width_2"
   ],
 
-  "particle_classification_features": [
+  "disp_classification_features": [
     "log_intensity",
     "width",
     "length",
-    "x",
-    "y",
-    "psi",
-    "phi",
     "wl",
     "skewness",
     "kurtosis",
-    "r",
     "time_gradient",
-    "leakage_intensity_width_2",
-    "log_reco_energy",
-    "reco_disp_dx",
-    "reco_disp_dy"
+    "leakage_intensity_width_2"
   ],
 
-  "disp_classification_features": [
+  "particle_classification_features": [
     "log_intensity",
     "width",
     "length",
-    "x",
-    "y",
```

Comment:
Is removing x and y really ok? We lose any info related to the image position on the camera...
In @misabelber's tests r, phi and psi were gone, but x and y were kept.

The one case in which I guess it may be good to omit them is in the disp classification, because if they are used, the training with inner gammas would certainly introduce a bias towards reconstructing background also close to the FOV center.

Reply:
ok, x and y are back now.

Reply:
I never used x and y for training of the disp regression / sign classification and I think when I tried it out it had even detrimental effects (less generalization). This should be easy to test though. I'd rather keep `r` and discard `x` and `y` for disp norm, intuitively.

Reply:
Now they are only in E-regression and particle classification, not on disp (though I actually meant to keep them in the disp regression - and not in classification - but that would also introduce some bias when training with inner-FOV-gammas, so let's keep it like it is now for the default).